### ISSUE TITLE
Mods with colon in name do not work in Tinker Time

### ISSUE THREAD

**Issue by oharaandrew314** [oharaandrew314/TinkerTime#176]
Example: [KOS](https://kerbalstuff.com/mod/86/kOS:%20Scriptable%20Autopilot%20System)

![proof](http://i.imgur.com/R4mmVrG.png)

This ticket is via UserVoice.  The error seems to occur when copying the mod to storage immediately after the download step of the import workflow.

Stacktrace:
```
java.nio.file.InvalidPathException: Illegal char <:> at index 3: kOS: Scriptable Autopilot System 0.15.5.zip
	at sun.nio.fs.WindowsPathParser.normalize(Unknown Source)
	at sun.nio.fs.WindowsPathParser.parse(Unknown Source)
	at sun.nio.fs.WindowsPathParser.parse(Unknown Source)
	at sun.nio.fs.WindowsPath.parse(Unknown Source)
	at sun.nio.fs.WindowsFileSystem.getPath(Unknown Source)
	at sun.nio.fs.AbstractPath.resolve(Unknown Source)
	at aohara.tinkertime.models.Mod.getCachedZipPath(Mod.java:75)
	at aohara.tinkertime.workflows.ModDownloaderContext.getDownloadPath(ModDownloaderContext.java:35)
	at aohara.tinkertime.workflows.tasks.MoveCrawlerDownloadToDestTask.call(MoveCrawlerDownloadToDestTask.java:38)
	at aohara.common.workflows.Workflow.run(Workflow.java:57)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
```

**Comment by apemanzilla** [comment]
Seems to be mostly an issue with Java in this case. It might be best just to replace colons with hyphens or similar when adding mods.

**Comment by oharaandrew314** [comment]
Probably

**Comment by Grayzzur** [comment]
Also doesn't work with slashes
https://kerbalstuff.com/mod/74/In%20game%20notes%20/%20notepad%20/%20checklist

I would guess any character in the name that's not a valid character in a windows/mac/unix filename would need to be handled.

**Comment by oharaandrew314** [comment]
Fixed in 99b9fd16a67490615bd81a77b7487c3f2d23cbd3
